### DISCUSSION TITLE
Download size exploded with 3.6.3 release!

### DISCUSSION THREAD

**Discussion by joshcsmith13** [openaudible/openaudible#1150] [General]
I was going to download the latest release (as of 3/1/2023), 3.6.3, but I ended up cancelling it after waiting an extraordinarily long time and realizing that it was 652MB!! 
The previous Win 64 installer, OpenAudible_3.6.2_x86_64.exe, was 147 MB, which still seems like a hefty amount.
This latest download, OpenAudible_3.6.3_x86_64.exe, has increased over 500 MB (.5 GB!)!   That can't be right. ??

**Comment by openaudible** [comment]
Good question. 
This was because we included what amounts to the chrome browser into version 3.6.3 release in an attempt to fix a bug that some Mac users on Intel were seeing (in code that we didn't write.) 
The beta is back to a good size if you wan to go there..  https://openaudible.org/beta or the next version 3.7 will be better.
Thanks for letting us know...
